pyccl/pyutils.py

```diff
@@ -570,3 +570,33 @@ def _get_spline2d_arrays(gsl_spline):
     check(status)
 
     return yarr, xarr, zarr.reshape(y_size, x_size)
+
+
+def _get_spline3d_arrays(gsl_spline, length):
+    """Get array data from a 3D GSL spline.
```

Comment:
An array of 2D GSL splines, instead of a 3D GSL spline?

Reply:
Done.